au4/src/projectscene/view/timeline/timelineruler.cpp

```diff
@@ -101,31 +101,31 @@ Ticks TimelineRuler::prepareTickData(const IntervalInfo& timeInterval, double w,
     {
         // determine tick type
         TickType tickType;
-        if (tickNumber % static_cast<int>(timeInterval.major / timeInterval.minorMinor) == 0) {
+        double eps = 1.0e-5f;
```

Comment:
better use std::numeric_limits<double>::epsilon()

Reply:
Yup, I thought that too but I tried that and it's to small... so I'm using the same value as AU3 does